pkg/apis/configuration/v1/types.go

```diff
@@ -652,82 +880,128 @@ type VariableCondition struct {
+	ClientSecret string `json:"clientSecret"`
+	// List of OpenID Connect scopes. The scope openid always needs to be present and others can be added concatenating them with a + sign, for example openid+profile+email, openid+email+userDefinedScope. The default is openid.
+	Scope string `json:"scope"`
+	// Allows overriding the default redirect URI. The default is /_codexch.
+	RedirectURI string `json:"redirectURI"`
+	// URL provided by your OpenID Connect provider to request the end user be logged out.
+	EndSessionEndpoint string `json:"endSessionEndpoint"`
+	// URI to redirect to after the logout has been performed. Requires endSessionEndpoint. The default is /_logout.
+	PostLogoutRedirectURI string `json:"postLogoutRedirectURI"`
+	// Specifies the maximum timeout in milliseconds for synchronizing ID/access tokens and shared values between Ingress Controller pods. The default is 200.
+	ZoneSyncLeeway *int `json:"zoneSyncLeeway"`
+	// A list of extra URL arguments to pass to the authorization endpoint provided by your OpenID Connect provider. Arguments must be URL encoded, multiple arguments may be included in the list, for example [ arg1=value1, arg2=value2 ]
+	AuthExtraArgs []string `json:"authExtraArgs"`
+	// Option of whether Bearer token is used to authorize NGINX to access protected backend.
+	AccessTokenEnable bool `json:"accessTokenEnable"`
+	// Switches Proof Key for Code Exchange on. The OpenID client needs to be in public mode. clientSecret is not used in this mode.
+	PKCEEnable bool `json:"pkceEnable"`
+}
+
+// The WAF policy configures NGINX Plus to secure client requests using App Protect WAF policies.
 type WAF struct {
-	Enable       bool           `json:"enable"`
-	ApPolicy     string         `json:"apPolicy"`
-	ApBundle     string         `json:"apBundle"`
-	SecurityLog  *SecurityLog   `json:"securityLog"`
+	// Enables NGINX App Protect WAF.
+	Enable bool `json:"enable"`
+	// The App Protect WAF policy of the WAF. Accepts an optional namespace. Mutually exclusive with apBundle.
+	ApPolicy string `json:"apPolicy"`
+	// The App Protect WAF policy bundle. Mutually exclusive with apPolicy.
+	ApBundle string `json:"apBundle"`
+	//
+	SecurityLog *SecurityLog `json:"securityLog"`
+	//
 	SecurityLogs []*SecurityLog `json:"securityLogs"`
 }
 
 // SecurityLog defines the security log of a WAF policy.
 type SecurityLog struct {
-	Enable      bool   `json:"enable"`
-	ApLogConf   string `json:"apLogConf"`
+	// Enables security log.
+	Enable bool `json:"enable"`
+	// The App Protect WAF log conf resource. Accepts an optional namespace. Only works with apPolicy.
+	ApLogConf string `json:"apLogConf"`
+	// The App Protect WAF log bundle resource. Only works with apBundle.
 	ApLogBundle string `json:"apLogBundle"`
-	LogDest     string `json:"logDest"`
+	// The log destination for the security log. Only accepted variables are syslog:server=<ip-address &#124; localhost; fqdn>:<port>, stderr, <absolute path to file>.
```

Comment:
Seems like an encoding error inherited from the docs.

Reply:
@mohamadaldawamnah lets correct the error here and we can open another PR in https://github.com/nginx/documentation/blob/main/content/nic/configuration/policy-resource.md?plain=1#L796 later cc @ADubhlaoich